aptible/database.go

```diff
@@ -188,7 +217,8 @@ func (c *Client) GetDatabase(databaseID int64) (Database, error) {
 func (c *Client) UpdateDatabase(databaseID int64, updates DBUpdates) error {
 	requestType := "restart"
 	request := models.AppRequest24{
-		Type: &requestType,
+		Type:            &requestType,
+		InstanceProfile: updates.ContainerProfile,
```

Comment:
if someone passes in a zero value here, it fails gracefully or is safely ignored, right?

Reply:
It is ignored (omitted).

```
InstanceProfile string `json:"instance_profile,omitempty"`
```

Reply:
in discussion this is probably safe because the backend hydrates this from an existing value, so customers who may have used this setting outside of terraform won't lose their settings